lib/transform/html.js

```diff
@@ -60,49 +60,56 @@ function transformDocument(file, config) {
-                    rebaseScriptTag(element, file, config);
-                    break;
-                case "LINK":
-                    if (config.cssEmbedding && element.getAttribute("rel") === "stylesheet") {
-                        href = element.getAttribute("href");
-                        if (href.indexOf(":") === -1) {
-                            cssFile = config.files[URL.resolve(file.location, href)];
-                            cssFile = cssFile ? cssFile.utf8 : "";
-                            styleElement = file.document.createElement("style");
-                            var appPackageRelativePath = URL.resolve(
-                                    href,
-                                    URL.relative(
-                                        appPackage.buildLocation,
-                                        file.buildLocation
-                                    )
-                                ),
-                                cssText;
-
-                            cssText = transformCss.resolve(appPackageRelativePath, cssFile, config).trim();
-                            if (cssText.length) {
-                                styleElement.appendChild(
-                                    file.document.createTextNode(cssText)
-                                );
-                                element.parentNode.replaceChild(styleElement, element);
-                            } else {
-                                element.parentNode.removeChild(element);
-                            }
+            case "HTML":
+                if (appPackage.packageDescription.appcache) {
+                    var relativeLocation = URL.relative(file.buildLocation, appcacheLocation);
+                    element.setAttribute("manifest", relativeLocation);
+                }
+                break;
+            case "STYLE":
+                rebaseStyleTag(element, file, config);
+                break;
+            case "SCRIPT":
+                rebaseAttribute(element, "src", file, config);
+                rebaseScriptTag(element, file, config);
+                break;
+            case "LINK":
+                if (config.cssEmbedding && element.getAttribute("rel") === "stylesheet") {
+                    href = element.getAttribute("href");
+                    if (href.indexOf(":") === -1) {
+                        cssFile = config.files[URL.resolve(file.location, href)];
+                        if (cssFile === undefined) {
+                            config.out.warn("Stylesheet file not found: " + URL.resolve(file.location, href));
+                        }
+                        cssFile = cssFile ? cssFile.utf8 : "";
+                        styleElement = file.document.createElement("style");
+                        rebaseAttribute(element, "href", file, config);
+                        var appPackageRelativePath = URL.resolve(
+                                element.getAttribute("href"),
```

Comment:
In fact this is part of the bug fix. The first href is now before rebase, this second time of getAttribute, the href is after rebase. It was done like that on purpose and it has to be like that